### PR TITLE
Validation feature

### PR DESCRIPTION
**Pull request by j-tee**
# In This Milestone:
- Add the following validations:
  - For the User model:
    - Name must not be blank.
    - PostsCounter must be an integer greater than or equal to zero.
  - For the Post model:
    - Title must not be blank.
    - Title must not exceed 250 characters.
    - CommentsCounter must be an integer greater than or equal to zero.
    - LikesCounter must be an integer greater than or equal to zero.
- Add unit specs for all models' methods and validations.

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @j-tee , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that still has the template logo present and the [MIT](https://github.com/j-tee/my-blog-app/blob/validation-feature/LICENSE) licensed link broken.

- [ ] Please remove the Microverse logo from the [README template](https://github.com/microverseinc/readme-template).
- [ ] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.

- [ ] Please fix [MIT](https://github.com/j-tee/my-blog-app/blob/validation-feature/LICENSE) licensed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/models/like.rb [hunk 1]
```diff
@@ -2,7 +2,10 @@ class Like < ActiveRecord::Base
   belongs_to :user, class_name: 'User'
   belongs_to :post, class_name: 'Post'
 
+  after_save :update_likes_for_posts
+
   def update_likes_for_posts
     post.likes_counter = post.likes.count
+    post.save
   end
 end
```
Comment (fobadara) [inline app/models/like.rb:11]: [REQUIRED] I can't find a test for this method. Kindly include tests for all custom methods

File: app/models/comment.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@ class Comment < ActiveRecord::Base
   belongs_to :user, class_name: 'User'
   belongs_to :post, class_name: 'Post'
 
+  validates :text, presence: true, length: { in: 1..500 }
+
+  after_save :update_comments_for_post
   def update_comments_for_post
     post.comments_counter = post.comments.count
   end
```
Comment (fobadara) [inline app/models/comment.rb]: [REQUIRED] I can't find a test for this method. Kindly include tests for all custom methods

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @j-tee, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @j-tee 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**